Content.Server/Flash/FlashSystem.cs

```diff
@@ -106,7 +107,7 @@ private bool UseFlash(EntityUid uid, FlashComponent comp, EntityUid user)
             return true;
         }
 
-        public void Flash(EntityUid target, EntityUid? user, EntityUid? used, float flashDuration, float slowTo, bool displayPopup = true, FlashableComponent? flashable = null)
+        public void Flash(EntityUid target, EntityUid? user, EntityUid? used, float flashDuration, float slowTo, bool displayPopup = true, FlashableComponent? flashable = null, bool melee = false)
         {
             if (!Resolve(target, ref flashable, false)) return;
```

Comment:
Melee shouldn't be a bool, just use user / used / target.

Reply:
I had to use this bool because otherwise whenever the flash is used normally it will end up converting anyone hit by it.

Reply:
It should be checking the user.

Reply:
No I mean if the user (who is head rev) uses the flash normally it will convert everyone hit by it without this check.

Reply:
Doesn't the user on flash args handle it?

Reply:
Yes?